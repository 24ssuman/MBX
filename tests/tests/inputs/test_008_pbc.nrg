SYSTEM 1H2O
MOLECULE
MONOMER h2o
O  48.40882694  51.07433216  50.0
H  49.36408029  50.9789852  50.0
H  48.08837235  51.97926799  50.0
ENDMON
ENDMOL
ENDSYS
SYSTEM 2H2O
MOLECULE
MONOMER h2o
O  48.41027575  51.04337922  49.9121916
H  49.36408029  50.9789852  50.0
H  48.0993372  51.7450105  49.3354501
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  51.64924507  51.08594656  50.0
H  52.60878026  51.09587704  49.97182885
H  51.33830653  51.78757784  50.5767415
ENDMON
ENDMOL
ENDSYS
SYSTEM 3H2O
MOLECULE
MONOMER h2o
O  48.41027575  51.04337922  49.9121916
H  49.36408029  50.9789852  50.0
H  48.0993372  51.7450105  49.3354501
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  51.64924507  51.08594656  50.0
H  52.60878026  51.09587704  49.97182885
H  51.33830653  51.78757784  50.5767415
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  49.38684791  52.46976336  52.07005086
H  50.34684791  52.46976336  52.07005086
H  49.06639333  53.37469919  52.07005086
ENDMON
ENDMOL
ENDSYS
SYSTEM 3H2O 1F
MOLECULE
MONOMER h2o
O  48.41027575  51.04337922  49.9121916
H  49.36408029  50.9789852  50.0
H  48.0993372  51.7450105  49.3354501
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  51.64924507  51.08594656  50.0
H  52.60878026  51.09587704  49.97182885
H  51.33830653  51.78757784  50.5767415
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  49.38684791  52.46976336  52.07005086
H  50.34684791  52.46976336  52.07005086
H  49.06639333  53.37469919  52.07005086
ENDMON
ENDMOL
MOLECULE
MONOMER f
F  48.67516528  55.35321954  52.07944727
ENDMON
ENDMOL
ENDSYS
SYSTEM 3H2O 1I
MOLECULE
MONOMER h2o
O  49.8075853152  51.915570533  50.2872950373
H  50.1309831146  51.516356055  49.4631224301
H  50.3660408759  51.460591995  50.9291320107
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  51.874700513  49.99555419161  51.263267096999996
H  52.706146075  50.4326890998  51.072348103
H  51.584433974  49.719085734  50.3718273463
ENDMON
ENDMOL
MOLECULE
MONOMER i
I  51.155633842  49.7031179641  48.07613641
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  48.044141015  49.547478256  49.6576903239
H  48.826432646  49.3253959686  49.1325643062
H  48.319132871  50.3896463053  50.03743062069
ENDMON
ENDMOL
ENDSYS
SYSTEM 5H2O 1Br
MOLECULE
MONOMER br
Br  47.7918688  50.1976693677  50.2393048227
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  51.321764984  49.90258957149  49.8233396599
H  51.417497992  50.6385845339  50.4439868776
H  50.3540663798  49.8411881269  49.7543926549
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  50.2944922158  47.648464251  51.44482416
H  50.9312334658  48.211619656  50.9860224962
H  49.5223238742  48.227827812  51.485185464
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  48.87716494  46.664446123  49.0201046064
H  49.4251932026  46.747763645  49.8106630492
H  48.461713427  47.532425101  48.980041967
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  50.1275636119  52.311277753  48.344453788
H  50.7574431466  51.582562474  48.365231726
H  49.3135316235  51.872099084  48.631816579
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  50.5998702944  52.360680451  51.273512544
H  49.7869270467  51.844059568  51.366935547
H  50.5104601172  52.693542675  50.3706992464
ENDMON
ENDMOL
ENDSYS
SYSTEM 3H2O 1Cl
MOLECULE
MONOMER h2o
O  47.2220211372  48.7861489631  49.0098357579
H  46.8969316576  48.8633991162  49.9129048844
H  48.1525237878  49.0635563349  49.1233409217
ENDMON
ENDMOL
MOLECULE
MONOMER cl
Cl  50.0000029477  50.0000014926  49.9999994979
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  52.777980263399996  48.7861460305  50.9901720656
H  53.1030657497  48.8633797566  50.087100109
H  51.8474760058  49.0635480556  50.8766661895
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  47.2220180927  51.2138365139  50.9901681838
H  46.8969352935  51.1366277718  50.0870932044
H  48.1525203921  50.9364284622  50.8766593013
ENDMON
ENDMOL
ENDSYS
SYSTEM 6H2O 1Li
MOLECULE
MONOMER li
Li  49.8221776227  50.154600637  48.937909595
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  51.747459421  48.526276626  48.670321661
H  52.244482277  48.20255494  47.916479219
H  51.068405585  47.866148367  48.869582505
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  50.7437266483  51.374506618  50.8222811708
H  51.596608055  50.9404188344  51.025129375
H  50.8461162064  52.307704932  51.009748652
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  52.890592593  49.8293889562  50.9309514918
H  52.724092431  49.2121311449  50.2072612857
H  53.324378747  49.3315177559  51.626237663
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  49.2414101724  47.71956891  49.4153828958
H  48.671834794  46.990017805  49.1660182709
H  48.88418278  48.094232816  50.2330230461
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  49.0872100354  51.371113394  47.074212431
H  49.597479527  51.990769868  46.550022
H  48.215168962  51.369663116  46.676100281000004
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  48.57025344  49.7625510636  51.123643104
H  49.2419129107  50.3935077755  51.423234644
H  47.774199221  49.93208263337  51.630258892
ENDMON
ENDMOL
ENDSYS
SYSTEM 4H2O 1Na
MOLECULE
MONOMER na
Na  50.2983563686  50.5824427983  49.7674913922
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  52.772167811  50.8286480412  51.34076528
H  52.824222375  50.599511762  52.272062594
H  53.657770806  50.6729223989  51.0032623
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  48.930434506  48.376047673  49.7402237176
H  48.362766278  48.571845267  48.987069177
H  48.674233216  47.510665538  50.05917302993
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  48.223417389  50.2121118287  47.714924081
H  48.39232373  50.03967313465  46.785307521
H  47.38757246  50.6851661022  47.725445814
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  51.155776529  52.947707568  50.4081835272
H  51.941023561  52.684200283  50.8998914589
H  51.120605744  53.903764893  50.4393509883
ENDMON
ENDMOL
ENDSYS
SYSTEM 5H2O 1K
MOLECULE
MONOMER k
K  50.7901673722  51.612907569  49.90051528306
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  52.659749315  53.74594209  49.7288494145
H  53.616805241  53.690985694  49.6910143186
H  52.475696479999996  54.686935731  49.6955279602
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  47.540378455  48.669231368  50.1655871795
H  47.149769433  48.301787645  50.9620766633
H  47.085779826  48.235334983  49.4395983285
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  52.386893449  46.85440925  50.04110325595
H  52.708974666  46.359886285  49.2855347432
H  52.772927905  46.426302414  50.8071809206
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  48.013514769  51.507013803  50.02183888559
H  47.244907536  52.07738767  50.02924719843
H  47.671466567  50.6053968042  50.07556847015
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  50.4652068297  48.922333461  50.03161011576
H  49.5658140412  48.585387352  50.08411808591
H  51.049675431  48.152902207  50.04062916646
ENDMON
ENDMOL
ENDSYS
SYSTEM 5H2O 1Rb
MOLECULE
MONOMER rb
Rb  49.97981747243  50.2175545596  50.07474007752
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  48.257145658  45.958476443  49.94771867207
H  48.720029777  45.118539222  49.94474085873
H  47.33126656  45.710405236  49.91471833065
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  54.338313118  53.239018103  50.01332516094
H  55.29258358  53.244280194  50.1117518319
H  54.101250766  54.157212715  49.8780506518
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  54.643662103  48.597593272  50.0002413636679
H  54.525628558  47.648049837  50.05345805045
H  55.59157685  48.723625067  49.92963354997
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  48.112392844  52.622454587  50.02521226412
H  48.343923559  53.553630083  50.005152148464
H  47.152583556  52.620863003  50.02635984769
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  52.784303808  50.7934250449  50.03193442325
H  53.444767752  50.0887674314  50.01951160996
H  53.294529665  51.613500481  50.02345115879
ENDMON
ENDMOL
ENDSYS
SYSTEM 5H2O 1Cs
MOLECULE
MONOMER cs
Cs  53.282293685  52.472547627  49.663142333
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  44.72813104  44.881561208  49.93555240783
H  44.957026354  44.02981214  50.3087563258
H  43.78634948  44.821618476  49.768084959
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  58.724746036  45.929275668  49.5059666187
H  58.584254966  44.981906207  49.4739279475
H  59.675007155  46.023517834  49.584782977
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  54.642386244  54.884658192  49.2068031319
H  55.588814045  55.043530554  49.1876929633
H  54.256517072  55.749582788  49.0509945205
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  51.957774296  51.649823825  47.319606137
H  52.279807956  51.053814129  46.639394581
H  51.074407823  51.883791633  47.025828763
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O  52.439353120999996  52.509743329  54.949655679
H  52.954676194  52.040718998  55.608232677000004
H  51.744502153  52.932339837  55.457527461
ENDMON
ENDMOL
ENDSYS
