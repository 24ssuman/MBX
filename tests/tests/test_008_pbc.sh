#!/bin/bash

filename=$(basename "$0")
filename="${filename%.*}"

mkdir -p outputs

../../install/bin/pbc-test inputs/${filename}.nrg > outputs/${filename}.out

