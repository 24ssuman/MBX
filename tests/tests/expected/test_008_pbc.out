system[.........0] e1b(nopbc)=    1.677857e+00  e1b(pbc)=    1.677857e+00    kcal/mol
system[.........0] e2b(nopbc)=    0.000000e+00  e2b(pbc)=    0.000000e+00    kcal/mol
system[.........0] e3b(nopbc)=    0.000000e+00  e3b(pbc)=    0.000000e+00    kcal/mol

system[.........1] e1b(nopbc)=    2.069254e+00  e1b(pbc)=    2.069254e+00    kcal/mol
system[.........1] e2b(nopbc)=    5.362417e-01  e2b(pbc)=    5.362417e-01    kcal/mol
system[.........1] e3b(nopbc)=    0.000000e+00  e3b(pbc)=    0.000000e+00    kcal/mol

system[.........2] e1b(nopbc)=    2.460651e+00  e1b(pbc)=    2.460651e+00    kcal/mol
system[.........2] e2b(nopbc)=    1.716837e+00  e2b(pbc)=    1.716837e+00    kcal/mol
system[.........2] e3b(nopbc)=    2.211225e-01  e3b(pbc)=    2.211225e-01    kcal/mol

system[.........3] e1b(nopbc)=    2.460651e+00  e1b(pbc)=    2.460651e+00    kcal/mol
system[.........3] e2b(nopbc)=    4.871826e+00  e2b(pbc)=    4.871826e+00    kcal/mol
system[.........3] e3b(nopbc)=    2.211225e-01  e3b(pbc)=    2.211225e-01    kcal/mol

system[.........4] e1b(nopbc)=    9.435641e-01  e1b(pbc)=    9.435641e-01    kcal/mol
system[.........4] e2b(nopbc)=    2.042973e+01  e2b(pbc)=    2.042973e+01    kcal/mol
system[.........4] e3b(nopbc)=   -2.158444e-02  e3b(pbc)=   -2.158444e-02    kcal/mol

system[.........5] e1b(nopbc)=    1.075945e+00  e1b(pbc)=    1.075945e+00    kcal/mol
system[.........5] e2b(nopbc)=    1.032743e+01  e2b(pbc)=    1.032743e+01    kcal/mol
system[.........5] e3b(nopbc)=   -3.274504e-01  e3b(pbc)=   -3.274504e-01    kcal/mol

system[.........6] e1b(nopbc)=    1.147357e+00  e1b(pbc)=    1.147357e+00    kcal/mol
system[.........6] e2b(nopbc)=    1.295413e+01  e2b(pbc)=    1.295413e+01    kcal/mol
system[.........6] e3b(nopbc)=    0.000000e+00  e3b(pbc)=    0.000000e+00    kcal/mol

system[.........7] e1b(nopbc)=    1.120987e+00  e1b(pbc)=    1.120987e+00    kcal/mol
system[.........7] e2b(nopbc)=    2.293930e+01  e2b(pbc)=    2.293930e+01    kcal/mol
system[.........7] e3b(nopbc)=    8.771741e-01  e3b(pbc)=    8.771741e-01    kcal/mol

system[.........8] e1b(nopbc)=    2.062706e-01  e1b(pbc)=    2.062706e-01    kcal/mol
system[.........8] e2b(nopbc)=    7.018067e+00  e2b(pbc)=    7.018067e+00    kcal/mol
system[.........8] e3b(nopbc)=    0.000000e+00  e3b(pbc)=    0.000000e+00    kcal/mol

system[.........9] e1b(nopbc)=    2.358216e-01  e1b(pbc)=    2.358216e-01    kcal/mol
system[.........9] e2b(nopbc)=    1.312984e+01  e2b(pbc)=    1.312984e+01    kcal/mol
system[.........9] e3b(nopbc)=    2.854534e-04  e3b(pbc)=    2.854534e-04    kcal/mol

system[........10] e1b(nopbc)=    1.042309e-01  e1b(pbc)=    1.042309e-01    kcal/mol
system[........10] e2b(nopbc)=    8.483600e+00  e2b(pbc)=    8.483600e+00    kcal/mol
system[........10] e3b(nopbc)=    1.115980e-01  e3b(pbc)=    1.115980e-01    kcal/mol

system[........11] e1b(nopbc)=    2.076022e-02  e1b(pbc)=    2.076022e-02    kcal/mol
system[........11] e2b(nopbc)=    1.066894e+01  e2b(pbc)=    1.066894e+01    kcal/mol
system[........11] e3b(nopbc)=    0.000000e+00  e3b(pbc)=    0.000000e+00    kcal/mol


  No cutoff dispersion energy:

  -5.071713e-04  -4.955629e-04  -4.842635e-04  -4.732639e-04  -4.625550e-04
  -4.521281e-04  -4.419746e-04  -4.320864e-04  -4.224556e-04  -4.130745e-04
  -4.039359e-04  -3.950324e-04  -3.863574e-04  -3.779039e-04  -3.696657e-04
  -3.616365e-04  -3.538102e-04  -3.461809e-04  -3.387431e-04  -3.314913e-04
  -3.244200e-04  -3.175243e-04  -3.107992e-04  -3.042398e-04  -2.978415e-04
  -2.915997e-04  -2.855102e-04  -2.795687e-04  -2.737710e-04  -2.681132e-04
  -2.625916e-04  -2.572022e-04  -2.519416e-04  -2.468061e-04  -2.417925e-04
  -2.368974e-04  -2.321176e-04  -2.274501e-04  -2.228918e-04  -2.184398e-04
  -2.140913e-04  -2.098435e-04  -2.056938e-04  -2.016396e-04  -1.976784e-04
  -1.938078e-04  -1.900254e-04  -1.863289e-04  -1.827160e-04  -1.791847e-04
  -1.757329e-04  -1.723584e-04  -1.690593e-04  -1.658337e-04  -1.626798e-04
  -1.595957e-04  -1.565796e-04  -1.536299e-04  -1.507448e-04  -1.479228e-04

  Cutoff dispersion energy:

  -2.937509e-04  -2.821424e-04  -2.708431e-04  -2.598435e-04  -2.491346e-04
  -2.387076e-04  -2.285541e-04  -2.186659e-04  -2.090351e-04  -1.996541e-04
  -1.905154e-04  -1.816120e-04  -1.729369e-04  -1.644835e-04  -1.562453e-04
  -1.482161e-04  -1.403898e-04  -1.327605e-04  -1.253227e-04  -1.180708e-04
  -1.109996e-04  -1.041039e-04  -9.737875e-05  -9.081935e-05  -8.507351e-05
  -7.980225e-05  -7.466097e-05  -6.964600e-05  -6.475382e-05  -5.998099e-05
  -5.532420e-05  -5.078023e-05  -4.634598e-05  -4.201843e-05  -3.779464e-05
  -3.367178e-05  -2.964712e-05  -2.571797e-05  -2.188177e-05  -1.813600e-05
  -1.447824e-05  -1.315308e-05  -1.185963e-05  -1.059703e-05  -9.364448e-06
  -8.161061e-06  -6.986086e-06  -5.838759e-06  -4.718338e-06  -3.624107e-06
  -2.555367e-06  -1.511443e-06  -4.916780e-07   0.000000e+00   0.000000e+00
   0.000000e+00   0.000000e+00   0.000000e+00   0.000000e+00   0.000000e+00


  Difference between cutoff and no cutoff dispersion energy:

   2.134204e-04   2.134204e-04   2.134204e-04   2.134204e-04   2.134204e-04
   2.134204e-04   2.134204e-04   2.134204e-04   2.134204e-04   2.134204e-04
   2.134204e-04   2.134204e-04   2.134204e-04   2.134204e-04   2.134204e-04
   2.134204e-04   2.134204e-04   2.134204e-04   2.134204e-04   2.134204e-04
   2.134204e-04   2.134204e-04   2.134204e-04   2.134204e-04   2.127680e-04
   2.117975e-04   2.108492e-04   2.099227e-04   2.090172e-04   2.081323e-04
   2.072674e-04   2.064220e-04   2.055956e-04   2.047877e-04   2.039979e-04
   2.032256e-04   2.024705e-04   2.017321e-04   2.010100e-04   2.003038e-04
   1.996130e-04   1.966904e-04   1.938342e-04   1.910426e-04   1.883140e-04
   1.856468e-04   1.830393e-04   1.804901e-04   1.779977e-04   1.755606e-04
   1.731775e-04   1.708469e-04   1.685676e-04   1.658337e-04   1.626798e-04
   1.595957e-04   1.565796e-04   1.536299e-04   1.507448e-04   1.479228e-04
